src/sonic-config-engine/tests/sample_output/l2switch.json

```diff
@@ -1,266 +1,304 @@
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/84", 
+            "lanes": "109,110,111,112"
+        }, 
         "Ethernet88": {
-            "alias": "fortyGigE0/88",
-            "lanes": "117,118,119,120",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/88", 
+            "lanes": "117,118,119,120"
+        }, 
         "Ethernet92": {
-            "alias": "fortyGigE0/92",
-            "lanes": "113,114,115,116",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/92", 
+            "lanes": "113,114,115,116"
+        }, 
         "Ethernet96": {
-            "alias": "fortyGigE0/96",
-            "lanes": "121,122,123,124",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/96", 
+            "lanes": "121,122,123,124"
+        }, 
         "Ethernet100": {
-            "alias": "fortyGigE0/100",
-            "lanes": "125,126,127,128",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/100", 
+            "lanes": "125,126,127,128"
+        }, 
         "Ethernet104": {
-            "alias": "fortyGigE0/104",
-            "lanes": "85,86,87,88",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/104", 
+            "lanes": "85,86,87,88"
+        }, 
         "Ethernet108": {
-            "alias": "fortyGigE0/108",
-            "lanes": "81,82,83,84",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/108", 
+            "lanes": "81,82,83,84"
+        }, 
         "Ethernet112": {
-            "alias": "fortyGigE0/112",
-            "lanes": "89,90,91,92",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/112", 
+            "lanes": "89,90,91,92"
+        }, 
         "Ethernet116": {
-            "alias": "fortyGigE0/116",
-            "lanes": "93,94,95,96",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/116", 
+            "lanes": "93,94,95,96"
+        }, 
         "Ethernet120": {
-            "alias": "fortyGigE0/120",
-            "lanes": "97,98,99,100",
-            "admin_status": "up"
-        },
+            "admin_status": "up", 
+            "alias": "fortyGigE0/120", 
+            "lanes": "97,98,99,100"
+        }, 
         "Ethernet124": {
-            "alias": "fortyGigE0/124",
-            "lanes": "101,102,103,104",
-            "admin_status": "up"
+            "admin_status": "up", 
+            "alias": "fortyGigE0/124", 
+            "lanes": "101,102,103,104"
         }
-    },
+    }, 
     "VLAN": {
         "Vlan1000": {
+            "members": [
```

Comment:
this is not needed. should get rid of this.

Reply:
Fixed.